notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<ul><li>scaled down Generative --&gt; scaled down generative</li><li>Maybe mention the number of books contained in simplebooks-92 to give an idea of the size of the dataset</li><li>This demonstrate --&gt; This notebook demonstrate</li><li>Add a "Learning objective" section to the overview with bullet points to match the style of the other notebooks (it seems the last sentence of this paragraph lists the learning objectives)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>